components/brave_rewards/browser/rewards_service_impl.cc

```diff
@@ -1931,6 +1931,11 @@ void RewardsServiceImpl::GetPublisherInfo(
     const std::string& publisher_key,
     GetPublisherInfoCallback callback) {
   if (!Connected()) {
```

Comment:
unrelated to the problem mentioned below, it seems like it would make more sense to have `MaybeStartProcess` which checks !Connected() and checks to make sure rewards is enabled